File: java-vertx/src/main/java/org/openapitools/server/api/model/JsonSuccessBaseAllOf.java
package org.openapitools.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonValue;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class JsonSuccessBaseAllOf   {
  


  public enum ResultEnum {
    SUCCESS("success");

    private String value;

    ResultEnum(String value) {
      this.value = value;
    }

    @Override
    @JsonValue
    public String toString() {
      return value;
    }
  }

  private ResultEnum result;
  private String msg;

  public JsonSuccessBaseAllOf () {

  }

  public JsonSuccessBaseAllOf (ResultEnum result, String msg) {
    this.result = result;
    this.msg = msg;
  }

    
  @JsonProperty("result")
  public ResultEnum getResult() {
    return result;
  }
  public void setResult(ResultEnum result) {
    this.result = result;
  }

    
  @JsonProperty("msg")
  public String getMsg() {
    return msg;
  }
  public void setMsg(String msg) {
    this.msg = msg;
  }


  @Override
  public boolean equals(Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    JsonSuccessBaseAllOf jsonSuccessBaseAllOf = (JsonSuccessBaseAllOf) o;
    return Objects.equals(result, jsonSuccessBaseAllOf.result) &&
        Objects.equals(msg, jsonSuccessBaseAllOf.msg);
  }

  @Override
  public int hashCode() {
    return Objects.hash(result, msg);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class JsonSuccessBaseAllOf {\n");
    
    sb.append("    result: ").append(toIndentedString(result)).append("\n");
    sb.append("    msg: ").append(toIndentedString(msg)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: kotlin-spring/src/main/kotlin/org/openapitools/model/JsonSuccessBaseAllOf.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonValue
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param result 
 * @param msg 
 */
data class JsonSuccessBaseAllOf(

    @field:JsonProperty("result", required = true) val result: JsonSuccessBaseAllOf.Result,

    @field:JsonProperty("msg", required = true) val msg: kotlin.String
) {

    /**
    * 
    * Values: success
    */
    enum class Result(val value: kotlin.String) {
    
        @JsonProperty("success") success("success");
    
    }

}

